include/triton/backend/backend_input_collector.h

```diff
@@ -201,7 +228,7 @@ class BackendInputCollector {
   size_t pending_copy_kernel_buffer_byte_size_;
   size_t pending_copy_kernel_buffer_offset_;
   size_t pending_copy_kernel_input_buffer_counts_;
-  RequestsList pending_copy_kernel_inputs_;
+  std::list<ContiguousBuffer> pending_copy_kernel_inputs_;
```

Comment:
pending_copy_kernel_inputs_ -> pending_copy_kernel_input_buffers_? 

Similarly, requests to request_buffers?